versions.json

```diff
@@ -1,6 +1,6 @@
 {
     "react": "8.56.0",
```

Comment:
Just for this time: since we have added back roosters-react, please bump this version to 9.0.0